Example/HorizonCalendarExample/HorizonCalendarExample/Demo View Controllers/DayRangeSelectionDemoViewController.swift

```diff
@@ -40,6 +40,16 @@ final class DayRangeSelectionDemoViewController: DemoViewController {
       }
 
       self.calendarView.setContent(self.makeContent())
+
+      if
+        UIAccessibility.isVoiceOverRunning,
+        let selectedDate = self.calendar.date(from: day.components)
+      {
+        self.calendarView.layoutIfNeeded()
```

Comment:
Forcing layout seems better than using DispatchQueue.main.async/asyncAfter here. It'll only happen when VO is running, and it guarantees that the view will be ready to receive accessibility focus.